meta/libraries.json

```diff
@@ -8,13 +8,15 @@
       "Bruno Lalande",
       "Mateusz Loskot",
       "Adam Wulkiewicz",
-      "Menelaos Karavelas"
+      "Menelaos Karavelas",
+      "Vissarion Fisikopoulos"
     ],
     "maintainers": [
       "Barend Gehrels <barend -at- xs4all.nl>",
       "Bruno Lalande <bruno.lalande -at- gmail.com>",
       "Mateusz Loskot <mateusz -at- loskot.net>",
-      "Adam Wulkiewicz <adam.wulkiewicz -at- gmail.com>"
+      "Adam Wulkiewicz <adam.wulkiewicz -at- gmail.com>",
+      "Vissarion Fisikopoulos <fisikop -at- gmail.com"
```

Comment:
Also here.

BTW I was not aware you were not yet listed here...

Reply:
me neither :)